components/public-api/typescript-common/package.json

```diff
@@ -17,16 +17,18 @@
     "build:cjs": "tsc",
     "build:esm": "tsc --module es2015 --outDir ./lib/esm",
     "test": "mocha './**/*.spec.js' --exclude './node_modules/**' --exclude './lib/esm/**' --exit",
+    "test:forceUpdate": "mocha './**/*.spec.js' --exclude './node_modules/**' --exclude './lib/esm/**' --exit -force -update",
```

Comment:
@mustard-mh it would be good to add README with how to do it and share on public api channel

Reply:
> one question will be how to iterate on a single test?

Do you mean to force update only one test? After exec this command, contributor and reviewer has responsibility to verify that golden files are expected. 

It's not just run it then that's ok, we want to have a constant output in converter